app/controllers/api/databases.php

```diff
@@ -79,12 +79,16 @@ function createAttribute(string $databaseId, string $collectionId, Document $att
         throw new Exception(Exception::DATABASE_NOT_FOUND);
     }
 
+    $events->setContext('database', $db);
```

Comment:
This context is required by some events so we have to make sure to set it before the DB operation so it's available in the handler.